### DISCUSSION TITLE
__next div does not fill entire viewport width on mobile

### DISCUSSION THREAD

**Discussion by dgh328** [vercel/next.js#48612] [Help]
### Summary

See here: https://stage.zarobky.pl/, display from mobile (or via dev tools).

I have set the body's background to green to make it clear. There is an extra green rectangle of space on the right. Tried a combination of setting body, __next div's widths to `100vw`, to no avail.

Has anyone encountered this issue? If so, how did you fix?

### Example

https://stage.zarobky.pl/

**Comment by SamKomesarook** [comment]
I am not able to see the extra green rectangle you mention, but I have encountered similar top-level css default issues, and using a css normalizer like [this](https://github.com/necolas/normalize.css) helped in those cases.
